python_ta/cfg/graph.py

```diff
@@ -335,15 +345,28 @@ def __init__(
         target: CFGBlock,
         edge_label: Optional[Any] = None,
         condition: Optional[NodeNG] = None,
+        negate: Optional[bool] = None,
     ) -> None:
         self.source = source
         self.target = target
         self.label = edge_label
         self.condition = condition
+        self.negate = negate
         self.source.successors.append(self)
         self.target.predecessors.append(self)
         self.z3_constraints = {}
 
+    def get_label(self) -> Optional[Any]:
```

Comment:
should be `Optional[str]`

Reply:
I wrote `Optional[Any]` here to be consistent with the type annotation of `self.label`. Would you want to me change both parts.

Reply:
Ah, great. Yes please change the instance attribute type as well